logfile.log

```diff
@@ -0,0 +1,42 @@
+2018-02-06 20:29:55  INFO [main] (PropertyLoader.java:46) - Start reading list of schemes...
```

Comment:
Are you really sure that this log should be included in the project?

Reply:
Oh, I'll clean

Reply:
Done